thenewboston_node/project/settings/templates/nginx.conf

```diff
@@ -16,6 +16,7 @@ server {
         proxy_pass http://node:8000/;  # the trailing slash '/' is important
 
         sub_filter '<!-- __BLOCKCHAIN_LINK_STUB__ -->' '<li><a href="/blockchain/">Blockchain</a></li>';
+        sub_filter '<!-- __BLOCKCHAIN_DOCS_LINK_STUB__ -->' '<li><a href="/blockchain-docs/">Blockchain Documentation</a></li>';
```

Comment:
Let's have something like:
`<li><a href="/blockchain-docs/">Blockchain Documentation</a> (<a href="/blockchain-docs/filename.rst">rst</a>) </li>`
and not use autoindex here